### ISSUE TITLE
can't serve static files!

### ISSUE THREAD

**Issue by rrlittle** [rrlittle/chat_dev_site#1]
templates are not serving static files! can't figure out what's going on with that!

**Comment by rrlittle** [comment]
I tried. removing the channels package and they started getting served! PROGRESS!!! 

I noticed in the game_site project channels is being imported higher up the chain of things. maybe that's the issue?

**Comment by rrlittle** [comment]
Nope. Tried reordering. I wonder what's going on?

**Comment by rrlittle** [comment]
Found this issue https://github.com/django/channels/issues/76 and it led to a solution!

I neglected to set up the channel_layer setting in the setting files. so Channels wasn't working prooperly. something to do with daphne or something. maybe by default it was bypassing the django template engine or something. 

no matter. adding this to the settings.py file fixed the issue
```python
CHANNEL_LAYERS = {
    "default":{
        "BACKEND": 'asgiref.inmemory.ChannelLayer',
        "ROUTING": 'chat_dev_site.routing.channel_routing',
    }
}
```
